Source/Scene/Cesium3DTilesetMetadata.js

```diff
@@ -15,55 +16,74 @@ import TilesetMetadata from "./TilesetMetadata.js";
  * </p>
  *
  * @param {Object} options Object with the following properties:
- * @param {String} options.extension The extension JSON object.
+ * @param {Object} [options.tilesetJson] The tileset JSON object. If this is undefined, then options.extension must be defined.
+ * @param {Object} [options.extension] The JSON object, used for backwards compatibility with the 3DTILES_metadata extension.
```

Comment:
Can these parameters be merged if at this level it's essentially the same schema? maybe call it `metadataJson`, then you either pass `metadataJson: tilesetJson` for 1.1 or `metadataJson: tilesetJson.extensions.3DTILES_metadata`